web/settings/passphrase.go

```diff
@@ -48,6 +50,13 @@ func registerPassphrase(c echo.Context) error {
 func updatePassphrase(c echo.Context) error {
 	instance := middlewares.GetInstance(c)
 
+	// Event if the current passphrase is sent in this request to work, we only
+	// allow enforce a valid permission to avoid having an unauthorized enpoint
+	// for bruteforce.
```

Comment:
This sentence seems to have some typos and is hard to read. Maybe something like:

> Even if the current passphrase is needed for this request to work, we enforce a valid permission to avoid having an unauthorized enpoint that can be bruteforced.

Reply:
Sorry about that... rewrote this comment without reading over...